src/Database/Log/QueryLogger.php

```diff
@@ -25,6 +26,34 @@
 class QueryLogger
 {
 
+    use InstanceConfigTrait;
+
+    /**
+     * Default Config
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'threshold' => 0,
+        'filter' => null
+    ];
+
+    /**
+     * Constructor
+     *
+     * The $config array takes the following keys:
+     *
+     * - threshold: Threshold in milliseconds to log only query running slower than the given threshold. Default is 0.
+     * - filter: A callable to filter based on the LoggedQuery object.
+     *
+     * @param array $config Config options
+     * @return void
+     */
+    public function __construct(array $config = [])
```

Comment:
Took a look but https://github.com/cakephp/docs/edit/3.0/en/core-libraries/logging.rst feels wrong and SlowQueryLogger doesn't exist and I am not familiar with the issue to be suited to create the docs.

Also it appears it is something that should - if possible - be setup in app.php over bootstrap.php?